LeapEngine/SceneGraph/GameObject.h

```diff
@@ -130,29 +140,27 @@ namespace leap
 		std::vector<std::unique_ptr<GameObject>> m_pChildrenToAdd{};
 		std::vector<std::unique_ptr<GameObject>> m_pChildren{};
 
-		std::vector<std::unique_ptr<Component>> m_pComponentsToAdd{};
-		std::vector<std::unique_ptr<Component>> m_pComponents{};
+		std::vector<ComponentInfo> m_pComponentsToAdd{};
```

Comment:
Remove the p from the variable names since they are no longer a collection of pointers